akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -96,8 +104,26 @@ object BackoffSupervisor {
     def getRef: Optional[ActorRef] = Optional.ofNullable(ref.orNull)
   }
 
-  private case object StartChild extends DeadLetterSuppression
-  private case class ResetRestartCount(current: Int) extends DeadLetterSuppression
+  final case object Reset
```

Comment:
Needs the same docs as the java API below.

```
Send this message to the [[BackoffSupervisor]] and it will reset the back-off.
This should be used in conjunction with `withManualReset` in [[BackoffOptions]].
```